### PR TITLE
bob: update tests to 1.2.0

### LINKED ISSUE #1147: bob: update tests and add version file
The bob tests should be updated to exactly match the canonical data. Also a version file should be added to match the canonical data version.

### PR DESCRIPTION
**Pull request by sjwarner-bp**
Fixes #1147

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/bob/src/test/java/BobTest.java [hunk 1]
```diff
@@ -115,25 +133,25 @@ public void shoutingWithUmlauts() {
```
Comment (FridaTveit) [inline exercises/bob/src/test/java/BobTest.java]: I think the `shoutingWithUmlauts` test has been removed as well? :)

**Comment by sjwarner-bp** [comment]
Another build failure... Different reason this time though!

```
There was an error(7).
Checking fallback: https://ftp.ruby-lang.org/pub/ruby/2.2/ruby-2.2.5.tar.bz2
No fallback URL could be found, try increasing timeout with:
    echo "export rvm_max_time_flag=20" >> ~/.rvmrc
There has been an error fetching the ruby interpreter. Halting the installation.
The command "rvm install 2.2.5" failed and exited with 7 during .
```

**Comment by sjwarner-bp** [comment]
And again...

```
+local build_path=/home/travis/build/exercism/java/build
+[[ -d /home/travis/build/exercism/java/build ]]
+cd exercises
+/home/travis/build/exercism/java/gradlew clean
Picked up _JAVA_OPTIONS: -Xmx2048m -Xms512m
Downloading https://services.gradle.org/distributions/gradle-4.0.1-bin.zip
Exception in thread "main" java.net.ConnectException: Connection timed out (Connection timed out)
```

Timeout errors?

**Comment by sjwarner-bp** [comment]
Passing now

**Comment by FridaTveit** [comment]
And very strange about the failing builds... It does look like timeout errors...

**Pull request merged**